PhysicsTools/PatAlgos/plugins/PATIsolatedTrackProducer.cc

```diff
@@ -0,0 +1,130 @@
+#include <string>
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include "DataFormats/PatCandidates/interface/IsolatedTrack.h"
+#include "DataFormats/Candidate/interface/CandidateFwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/Math/interface/deltaR.h"
+#include "DataFormats/Common/interface/RefToPtr.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "PhysicsTools/PatUtils/interface/MiniIsolation.h"
+
+namespace pat {
+
+    class PATIsolatedTrackProducer : public edm::EDProducer {
```

Comment:
this should be of a type known to threaded framework
https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkModuleTypes
edm::global would be a better choice if implementation allows, edm::stream is OK.
edm::one is rather an exception and will need some justification.